apps/openmw/mwmechanics/aifollow.cpp

```diff
@@ -171,6 +213,16 @@ std::string AiFollow::getFollowedActor()
     return mActorRefId;
 }
 
+void AiFollow::turnActorToFaceTarget(const osg::Vec3f& actorPosition, const osg::Vec3f& targetPosition, AiFollowStorage& storage)
+{
+    osg::Vec3f dir = targetPosition - actorPosition;
+
+    float faceAngleRadians = std::atan2(dir.x(), dir.y());
```

Comment:
We will get NaN here if direction is zero (actor and target in the same position)